angular/example/package.json

```diff
@@ -1,22 +1,17 @@
 {
   "name": "@ondewo/angular-proto-compiler-example",
-  "version": "0.1.0",
-  "description": "Example for how to use the ondewo proto angular compiler",
+  "version": "2.0.0",
```

Comment:
shouldn't it be `1.1.0`?

Reply:
this is just an example so it doesn't matter in the end